### PR TITLE
[new] Add material-design-icons

### PR DESCRIPTION
**Pull request by sitic**
https://github.com/google/material-design-icons

Note that this only adds the icon font (see http://google.github.io/material-design-icons/#icon-font-for-the-web).

### CODE REVIEW

File: ajax/libs/material-design-icons/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  ],
+  "author": "Material Design Authors",
+  "license": "CC-BY-4.0",
+  "bugs": {
+    "url": "https://github.com/google/material-design-icons/issues"
+  },
+  "homepage": "https://github.com/google/material-design-icons",
+  "autoupdate": {
+    "source": "git",
+    "target": "git@github.com:google/material-design-icons.git",
```
Comment (PeterDaveHello) [inline ajax/libs/material-design-icons/package.json]: Should use git protocol here.

File: ajax/libs/material-design-icons/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "basePath": "/",
```
Comment (PeterDaveHello) [inline ajax/libs/material-design-icons/package.json]: What about set basePath `iconfont`

File: ajax/libs/material-design-icons/package.json (hunk 1 above)
Comment (sitic) [inline ajax/libs/material-design-icons/package.json]: fixed

File: ajax/libs/material-design-icons/package.json (hunk 2 above)
Comment (sitic) [inline ajax/libs/material-design-icons/package.json]: I've changed it to `"basePath": "/iconfont/"`. I wasn't sure whether to do add it, as the material-design-icons also has a lot of sprite sheets in [/sprites/](https://github.com/google/material-design-icons/tree/master/sprites). I didn't want to add them as the iconfont is the primary method and the sprite sheets are both large in number and partially outdated, but leave the option open to include more files from different directories.

**Comment by sitic** [comment]
Please read the inline comment/question regarding `basePath` before merging :-)

**Comment by PeterDaveHello** [comment]
Please squash them into only one commit, thanks.

**Comment by sitic** [comment]
@PeterDaveHello they are already squashed, https://github.com/cdnjs/cdnjs/pull/4746/commits shows me only the 6ae8801 commit.

**Comment by PeterDaveHello** [comment]
Oops, sorry, but I think there are more related files should be included, what do you think?

**Comment by Piicksarn** [comment]
ping @cdnjs/intern @cdnjs/intern2

**Comment by pvnr0082t** [comment]
`material-design-icons` had already been merged in cc90504c1db7731a292ffc47bfb9d6687327b84e. I'm going to close this PR. Thanks for your contribution. @sitic

**Pull request closed without merging**